java/runtime/src/main/java/io/ray/runtime/context/LocalModeWorkerContext.java

```diff
@@ -48,23 +48,27 @@ public JobId getCurrentJobId() {
   @Override
   public ActorId getCurrentActorId() {
     TaskSpec taskSpec = currentTask.get();
-    if (taskSpec == null) {
-      return ActorId.NIL;
-    }
+    Preconditions.checkNotNull(
+        taskSpec,
+        "Current task is not set. It might you invoked this API in a user thread, which is not allowed.");
     return LocalModeTaskSubmitter.getActorId(taskSpec);
   }
 
   @Override
   public TaskType getCurrentTaskType() {
     TaskSpec taskSpec = currentTask.get();
-    Preconditions.checkNotNull(taskSpec, "Current task is not set.");
+    Preconditions.checkNotNull(
+        taskSpec,
+        "Current task is not set. It might you invoked this API in a user thread, which is not allowed.");
     return taskSpec.getType();
   }
 
   @Override
   public TaskId getCurrentTaskId() {
     TaskSpec taskSpec = currentTask.get();
-    Preconditions.checkState(taskSpec != null);
+    Preconditions.checkNotNull(
+        taskSpec,
+        "Current task is not set. It might you invoked this API in a user thread, which is not allowed.");
```

Comment:
```suggestion
        "Current task is not set. Maybe you invoked this API in a user-created thread not managed by Ray. Invoking this API in a user-created thread is not supported yet in local mode. You can switch to cluster mode.");
```

Better to create a helper method. The message is pretty long.